snippets/go.json

```diff
@@ -160,6 +160,10 @@
 			"prefix": "bf",
 			"body": "func Benchmark$1(b *testing.B) {\n\tfor ${2:i} := 0; ${2:i} < b.N; ${2:i}++ {\n\t\t$0\n\t}\n}"
 		},
+		"table tests": {
```

Comment:
Since the `test function` and the `benchmark function` use the prefix `tf` and bf` respectively, we should probably name this snippet `table test function` and use the prefix `tbd` instead?